demo.html

```diff
@@ -0,0 +1,35 @@
+<html>
+<head lang="en">
+    <meta charset="utf-8">
+</head>
+<body>
+    <form id="checkout-selection" method="POST">
+        <input type="radio" name="checkout" value="automatic">Automatic Checkout Demo<br>
```

Comment:
I agree. But for the future I would like to design the app a lot better, would make for a cool test app. Adding radio buttons would leave things open ended for the future.